change/@ni-nimble-components-90bda8e4-208e-4e0c-ae48-b3730a529c57.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I have added `@tiptap/extension-hard-break` package for force line break and will have keyboard accessibility `Ctrl/Shift/Cmd + Enter`, So It needs to be updated in the spec file.

Whether we can update the spec file in this same PR or separate PR is required? @jattasNI @rajsite